packages/yoga/src/Datepicker/web/Calendar.jsx

```diff
@@ -380,12 +390,12 @@ function Calendar({
           fill="primary"
           data-testid="previous-month-arrow"
         />
-        <Text.Body2 style={{ alignSelf: 'center' }} bold>
+        <Month style={{ alignSelf: 'center' }} bold>
```

Comment:
WDYT of adding this style `alignSelf: 'center'` to the `Month` component, so style object isn't recreated in every render?